addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/ExternalRedirectScanRule.java

```diff
@@ -101,13 +102,18 @@ public class ExternalRedirectScanRule extends AbstractAppParamPlugin
     /** The various (prioritized) payload to try */
     private static final String[] REDIRECT_TARGETS = {
         REDIRECT_SITE,
-        HttpHeader.SCHEME_HTTP + REDIRECT_SITE,
         HttpHeader.SCHEME_HTTPS + REDIRECT_SITE,
         HttpHeader.SCHEME_HTTPS + REDIRECT_SITE.replace(".", "%2e"), // Double encode the dots
         "5;URL='https://" + REDIRECT_SITE + "'",
         "URL='http://" + REDIRECT_SITE + "'",
-        HttpHeader.SCHEME_HTTP + "\\" + REDIRECT_SITE,
+        // Simple allow list bypass, ex: https://evil.com?dest=original_value
+        // Where "original_value" is whatever the parameter value initially was, ex:
+        // https://good.expected.com
+        HttpHeader.SCHEME_HTTPS + REDIRECT_SITE + "/?dest=" + ORIGINAL_VALUE_PLACEHOLDER,
```

Comment:
Where's this `dest` coming from?

Reply:
Random param. Happy to use a different name.

Reply:
Why use a name at all?

Reply:
Just seemed most logical to me

Reply:
I guess it could be a hashtag # or just ?blahblahblah

Reply:
Let me know what you prefer.

Reply:
The latter sounds fine to me, the content just needs to be there not be a value of any parameter.